static/app/views/issueDetails/traceTimeline/traceTimelineEvents.tsx

```diff
@@ -168,7 +168,7 @@ function NodeGroup({
                 groupEvents
                   .slice(0, 5)
                   .map(groupEvent =>
-                    'event.type' in groupEvent ? (
+                    'event.type' in ['error', 'default'] ? (
```

Comment:
Before, `event.type` would only be a field for events from the Discover dataset.

Reply:
Hmm this line will always return false as written. I think you want `['error', 'default'].includes(groupEvent.['event.type'])` instead

Reply:
I have no idea how I wrote that line of code!!

Anyway, I fixed it.